docs/src/DatePicker.doc.js

```diff
@@ -0,0 +1,398 @@
+        type: 'string',
+        description:
+          'Accessibility label for the next-month button on the calendar view that clients such as VoiceOver will read to describe the element.',
+      },
+      {
+        name: 'accessibilityLabelPreviousMonth',
+        required: true,
+        type: 'string',
+        description:
+          'Accessibility label for the previous-month button on the calendar view that clients such as VoiceOver will read to describe the element.',
+      },
+      {
+        name: 'helperText',
+        type: 'Node',
+        description:
+          'More information about how to complete the date picker field',
+        href: 'helperText',
+      },
+      { name: 'disabled', type: 'boolean', href: 'disabled' },
+      {
+        name: 'endDate',
+        type: 'Date',
+        description:
+          'End date for which selection is enabled. Required for date range picker.',
+        href: 'rangePicker',
+      },
+      { name: 'errorMessage', type: 'Node', href: 'errorMessage' },
+      {
+        name: 'excludeDates',
+        type: 'Array<Date>',
+        description: 'Array of disabled dates for selection',
+      },
+      { name: 'id', required: true, type: 'string' },
+      {
+        name: 'idealDirection',
+        type: `'up'|'right'|'down'|'left'`,
+        description: 'Preferred direction for the calendar flyout to open',
+      },
+      {
+        name: 'includeDates',
+        type: 'Array<Date>',
+        description: 'Array of enabled dates for selection',
+      },
+      {
+        name: 'isRTL',
+        type: 'boolean',
+        defaultValue: false,
+        description:
+          'isRTL (Right-To-Left) provides support to specific locales by positioning textfield icon accordingly',
+      },
+      { name: 'label', type: 'string' },
+      {
+        name: 'localeData',
+        type: 'date-fns locale objects',
+        description: `DatePicker accepts imported locales from the open source date utility library date-fns. To use 'localeData', first, add the date-fns dependency in package.json. Second, use static or dynamic imports to get the locales from date-fns. Then, pass the locale to the localeData prop.`,
+        href: 'localeData',
+      },
+      {
+        name: 'maxDate',
+        type: 'Date',
+        description: 'End date for which range selection is enabled',
+      },
+      {
+        name: 'minDate',
+        type: 'Date',
+        description: 'Start date for which range selection is enabled',
+      },
+      {
+        name: 'onChange',
+        required: true,
+        type: '({event: SyntheticInputEvent<>, value: Date }) => void',
+      },
+      { name: 'placeholder', type: 'string' },
+      {
+        name: 'selectsEnd',
+        type: 'boolean',
+        description:
+          'Defines a date field input to select the end date in a date range. Required for date range picker.',
+        href: 'rangePicker',
+      },
+      {
+        name: 'selectsStart',
```

Comment:
@AlbertCarreras let's change this to an enum instead: `select: 'start' | 'end'` which will combine `selectsEnd` and `selectsStart`

Reply:
fixed